shared/src/test/scala/coop/rchain/shared/StreamTSpec.scala

```diff
@@ -58,6 +58,16 @@ class StreamTSpec extends FunSpec with Matchers with GeneratorDrivenPropertyChec
 
     }
 
+    it(
+      "should allow taking the longest prefix of this StreamT whose elements satisfy the predicate") {
```

Comment:
Did you run casper/test or did the formatter do this?

Reply:
I run the shared/test, and the formatter did this.